question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -810,6 +810,42 @@ def matlab_to_numpy(a):
             return (None, 'Invalid format (missing square brackets and not a real number).')
 
 
+def latex_array_from_numpy_array(A, presentation_type='f', digits=2):
```

Comment:
Maybe call this `latex_from_2darray` to be consistent with `string_from_2darray`?